_includes/sections/video-streaming.html

```diff
@@ -0,0 +1,27 @@
+<h1 id="Video Streaming" class="anchor"><a href="#video"><i class="fas fa-link anchor-icon"></i></a> Video Streaming</h1>
+
+  <div class="row mb-2">
+
+    {% include card.html color="success"
+    title="MediaGoblin"
+    image="/assets/img/tools/mediagoblin.png"
+    url="https://www.mediagoblin.org/"
+    footer="OS: Windows, macOS, Linux, FreeBSD, Web."
+    description="MediaGoblin is a free/libre tool to host media files in a distributed manner."
+    %}
+
+    {% include card.html color="primary"
+    title="Invidious"
+    image="/assets/img/tools/invidious.png"
+    url="https://github.com/omarroth/invidious/wiki/Invidious-Instances"
+    footer="OS: Windows, macOS, Linux, Web."
+    description="Invidious, an open-source privacy respecting API for YouTube."
+    %}
+
+    </div>
+
+<h3>Worth Mentioning</h3>
+
+<ul>
+  <li><a href="https://freetubeapp.io/">FreeTube</a> - FreeTube is an free software video streaming client utilizing <a href='https://invidio.us/'>Invidious</a></li>
```

Comment:
It is worth mentioning that one of the key features of FreeTube is that it stores user data on the client and not on a server.

```suggestion
  <li><a href="https://freetubeapp.io/">FreeTube</a> - FreeTube is an open-source video streaming client utilizing the <a href='https://invidio.us/'>Invidious API</a>, letting you watch YouTube privately on desktop.  All of your user data is stored locally and never sent or published to the internet.</li>
```